Algorithms/RunningSumArray.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Algorithms
+{
+    public class RunningSumArray
+    {
+        /// <summary>
+        /// Big O
+        /// Complejidad: O(n) - El arreglo solo se recorre una unica vez y se le resta la posición 
+        /// anterior que ya tiene el ultimo valor acumulado
+        /// </summary>
+        /// <param name="nums">Array de enteros</param>
```

Comment:
Good coding practices:
If the parameter is called `nums` it can be anything, and then you end up with documentation like this one `Array de enteros`, here are two things:
- Imagine your code is international, it is always a good practice to do your code in a "neutral" language as english so everyone can understand it
- What could be a better variable naming? `numArray` for example and that could give you more information without reading the documentation.


Apply the same to other parts of the code, for example in shuffleArray, what's `l`, `j` ?

Reply:
OK, So i'll change comments.